src/aosm/azext_aosm/cli_handlers/onboarding_base_handler.py

```diff
@@ -121,6 +140,19 @@ def _read_config_from_file(self, input_json_path: Path) -> dict:
 
         return config_dict
 
+    def _read_params_config_from_file(self, input_json_path) -> dict:
+        """ Reads input file, takes only the {parameters:values} + returns config as dictionary"""
+        with open(input_json_path, "r", encoding="utf-8") as _file:
+            params_schema = json.load(_file)
+
+        santised_schema = {}

```

Comment:
typo: `santised` should be `sanitised` (use 'rename' refactor tool (F2 in VS Code), as this variable is referenced below.